tests/unit/records/test_api_views.py

```diff
@@ -141,3 +141,30 @@ def test_custom_search(es, api, json_headers, record_with_bucket,
                         custom=query),
                 headers=json_headers)
             assert len(res.json) == result
+
+
+@pytest.mark.parametrize(('query', 'result'), [
+    ("true", "true"),
+    ("True", "true"),
+    ("1", "true"),
+    ("1234", "1234"),
+    ("test", "test"),
```

Comment:
I would add a case where the value is `None`, to check the old behavior of `?all_versions`

Reply:
In the case where the value is `None` what should be the output?

Reply:
We treat `None` as "truthy", so it should return `"true"`

Reply:
Ok and I suppose it goes as `None` and not as `"None"`